### PR TITLE
added relevant packages listing to ecosystem

### PR DESCRIPTION
**Pull request by 0xMuluh**
ping: #604

### CODE REVIEW

File: inst/assets/bibliography.bib [hunk 1]
```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
...
+    title = "{Assessment of human microbiota stability across longitudinal samples using iteratively growing-partitioned clustering}",
+    journal = {Briefings in Bioinformatics},
+    volume = {23},
+    number = {2},
+    pages = {bbac055},
+    year = {2022},
+    month = {02},
+    issn = {1477-4054},
+    doi = {10.1093/bib/bbac055},
+    url = {https://doi.org/10.1093/bib/bbac055},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: you can remove the URLs when DOI is already provided

Comment (antagomir) [inline inst/assets/bibliography.bib]: you can remove the URLs when DOI is already provided

File: inst/assets/bibliography.bib [hunk 2]
```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
   doi = {https://doi.org/10.1038/nmeth.2897},
   url = {https://www.nature.com/articles/nmeth.2897},
   author = {Costea, Paul I. and Zeller, Georg and Sunagawa, Shinichi and Bork, Peer}
+}
+
+@article{Sanchez2022,
+    author = {Sánchez-Sánchez, Pedro and Santonja, Francisco J and Benítez-Páez, Alfonso},
+    title = "{Assessment of human microbiota stability across longitudinal samples using iteratively growing-partitioned clustering}",
+    journal = {Briefings in Bioinformatics},
```
Comment (antagomir) [inline inst/assets/bibliography.bib:2252]: Use double brackets to keep capitalization {{}}

File: inst/assets/bibliography.bib [hunk 3]
```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
...
+    month = {02},
+    issn = {1477-4054},
+    doi = {10.1093/bib/bbac055},
+    url = {https://doi.org/10.1093/bib/bbac055},
+    eprint = {https://academic.oup.com/bib/article-pdf/23/2/bbac055/42806514/bbac055.pdf}
+}
+
+@article{Nguyen2022,
+    author = {Nguyen QP, Hoen AG, Frost HR },
+    title = "{CBEA: Competitive balances for taxonomic enrichment analysis}",
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: use double brackets to force capitalization

File: inst/assets/bibliography.bib [hunk 4]
```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
...
+    journal = {BPLoS Comput Biol},
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: is the name correct here..?

File: inst/assets/bibliography.bib [hunk 5]
```diff
@@ -2244,4 +2244,55 @@ @article{Costea2014
...
+    volume = {18},
+    number = {5},
+    url = {ttps://doi.org/10.1371/journal.pcbi.1010091}
```
Comment (antagomir) [inline inst/assets/bibliography.bib]: error?

Comment (antagomir) [reply inst/assets/bibliography.bib]: use doi instead of url when possible

**Review by antagomir — CHANGES_REQUESTED** [review]
check feedback

File: inst/assets/bibliography.bib (hunk 1 above)
Comment (0xMuluh) [reply inst/assets/bibliography.bib]: ACK

**Review by antagomir — APPROVED** [review]

**Pull request merged**